runtime/src/accounts_db.rs

```diff
@@ -191,6 +191,8 @@ pub struct AccountStorageEntry {
     ///  any accounts in it
     /// status corresponding to the storage, lets us know that
     ///  the append_vec, once maxed out, then emptied, can be reclaimed
+    /// Currently serialized count isn't used at all. Rather, we recalculate it
```

Comment:
I don't quite understand this comment

Reply:
Oops, I've forgot to comment about this. You're right. No wonder to be confused.

First, this comment isn't related to this PR at all.
I added this comment to properly document the current implementation reality, which have changed over the course of prior snapshot stability PRs. I'm sure @sakridge can understand this comment, who shares the needed context.

Specifically this comment is referring to [this change](https://github.com/solana-labs/solana/pull/7010/files#diff-2099c5256db4eb5975c8834af38f6456R1140)

@sakridge what do you think fixing this before strict ABI audit thing will be under the effect? Or, at the least, we should create an issue. After all, it seems we can reconstruct this information when ingesting snapshots without problems.

Reply:
Yea, we can create an issue and we can move this comment out of this PR.

Reply:
Created! https://github.com/solana-labs/solana/issues/7442